Gemfile

```diff
@@ -3,6 +3,8 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.3.4'
 
+gem 'dotenv-rails', groups: [:development, :test]
```

Comment:
Cleaner to move this into the `development, test` block below

Reply:
I actually put it there first but the GitHub page instruction was to put it at the top.